app/views/spree/admin/pages/edit.html.erb

```diff
@@ -1,13 +1,35 @@
+<%= render 'spree/admin/shared/pages_tabs' %>
+
+<% admin_breadcrumb(t('spree.settings')) %>
 <% admin_breadcrumb(link_to plural_resource_name(Spree::Page), spree.admin_pages_path) %>
-<% admin_breadcrumb(t('spree.static_content.new_page')) %>
+<% admin_breadcrumb(@page.title) %>
+
+<% content_for :page_title do %>
+  <%= Spree.t(:editing_page) %>
+<% end %>
 
-<% content_for :page_actions do %><% end %>
+<% content_for :page_actions do %>
+  <li>
+    <%= button_link_to I18n.t("spree.back_to_pages"), spree.admin_pages_path, icon: 'arrow-left' %>
```

Comment:
I would remove the `I18n` namespace from views so we can leverage the more feature loaded version from`ActionView::Helpers::TranslationHelper`

Reply:
@spaghetticode good call, I'll address this separately in a repo-wide PR 👍